Changelog.md

```diff
@@ -2,6 +2,11 @@
 
 This file only reflects the changes that are made in this image. Please refer to the upstream GitLab [CHANGELOG](https://gitlab.com/gitlab-org/gitlab-foss/blob/master/CHANGELOG.md) for the list of changes in GitLab.
 
+**15.5.2**
+
+- gitlab: upgrade CE to v15.5.2
+- gitaly: upgrade to v15.5.2
+
```

Comment:
Could you be so kind and add

- ubuntu: upgrade to focal-20220922

(See my comment related to the Dockerfile.)